tests/unit-tests/jobs/helpers/jobVerifier-test.js

```diff
@@ -203,22 +203,22 @@ describe('Jobs verifier tests', function () {
         });
         it('if chaos experiments mentioned in the job exist, should pass', async () => {
             configHandlerStub.withArgs(consts.CONFIG.JOB_PLATFORM).resolves('KUBERNETES');
-            req = { body: { run_immediately: true, cron_expression: '* * * * *', enabled: true, experiments: [{ experiment_id: '1234', start_after: 1000 }] } };
+            req = { body: { run_immediately: true, cron_expression: '* * * * *', enabled: true, experiments: [{ experiment_id: '1234', start_after: 60 }] } };
             getChaosExperimentsByIdsStub.resolves([{ id: '1234' }]);
             await jobVerifier.verifyExperimentsExist(req, res, nextStub);
             should(nextStub.args[0][0]).eql(undefined);
         });
         it('if chaos experiments mentioned in the job do not exist, should fail', async () => {
             configHandlerStub.withArgs(consts.CONFIG.JOB_PLATFORM).resolves('KUBERNETES');
-            req = { body: { run_immediately: true, cron_expression: '* * * * *', enabled: true, experiments: [{ experiment_id: '1234', start_after: 1000 }] } };
+            req = { body: { run_immediately: true, cron_expression: '* * * * *', enabled: true, experiments: [{ experiment_id: '1234', start_after: 60 }] } };
             getChaosExperimentsByIdsStub.resolves([]);
             await jobVerifier.verifyExperimentsExist(req, res, nextStub);
             should(nextStub.args[0][0].message).eql('One or more chaos experiments are not configured. Job can not be created');
             should(nextStub.args[0][0].statusCode).eql(400);
         });
         it('if job platform is not KUBERNETES and  job has experiments job does not have experiments, should pass', async () => {
             configHandlerStub.withArgs(consts.CONFIG.JOB_PLATFORM).resolves('DOCKER');
-            req = { body: { run_immediately: true, cron_expression: '* * * * *', enabled: false, experiments: [{ experiment_id: '1234', start_after: 1000 }] } };
+            req = { body: { run_immediately: true, cron_expression: '* * * * *', enabled: false, experiments: [{ experiment_id: '1234', start_after: 60 }] } };
```

Comment:
wonder if it makes sense in all this test to make it 60 sec in stead of 1000ms=1sec

Reply:
it's just a placeholder these tests are not running experiments as job platform is not KUBERNETES